armada/handlers/tiller.py

```diff
@@ -169,6 +168,29 @@ def _pre_update_actions(self, actions, namespace):
         except Exception:
             LOG.debug("PRE: Could not create anything, please check yaml")
 
+    def delete_resource(self, release_name, resource_name, resource_type,
```

Comment:
@gardlt did you want me to implement some sort of wait for the resource to be deleted?